### ISSUE TITLE
Relevance according to modern versions of VS Code

### ISSUE THREAD

**Issue by john-cj** [Microsoft/vscode-backspace#6]
Hi, Johannes!

Thank you for this extension. There are a lot of positive reviews on Microsoft Marketpalce about it. However, I don't really sure is it still necessary for modern versions of VS Code. Here are my settings:

(I started to use VS Code only 2 weeks ago).

``` json
{
    "editor.autoIndent": true,
    "editor.detectIndentation": true,
    "editor.insertSpaces": true,
    "editor.renderWhitespace": "boundary"
}
```

When I launch VS Code without any extensions, it works (from my point of view) exactly as shown on animated GIF of Backspace++.

So, in other words, it seems the extension doesn't do anything (from my point of view, but I don't sure).

Probably I missed something? Cheers, John.

**Comment by jrieken** [comment]
Yeah, sad but true. I will retire the extension

**Comment by Harakku** [comment]
I started using vscode recently and encountered this years-old issue: https://github.com/Microsoft/vscode/issues/15598

To prevent the wrong indentation described in the issue, `editor.useTabStops` had to be disabled. After looking for ways to get the tab-stop functionality back, it seems that the only easy solution is this plugin.

So cheers, I guess.
